packages/esm-patient-queues-app/src/ugandaemr-configuration-overrrides.json

```diff
@@ -1,31 +1,11 @@
 {
-  "@openmrs/esm-patient-chart-app": {
-    "displayConditions":{
-      "privileges":[
-        "App: ugandaemrpoc.triage"
-      ]
-    },
-    "extensionSlots": {
-      "patient-actions-slot":{
-
-      },
-      "action-menu-chart-items-slot":{
-
-      },
+  "@openmrs/esm-patient-chart-app": { 
```

Comment:
These files need to be deleted so that configurations are loaded from one central place. Will do this in my branch after rebase

Reply:
ok